### DISCUSSION TITLE
Mockoon in docker have swagger-ui?

### DISCUSSION THREAD

**Discussion by grnmeadow7** [mockoon/mockoon#1154] [General]
Hi Everyone,

I am new in Mockoon. Is there a default swagger-ui for Mockoon project deployed in docker?

**Comment by 255kb** [comment]
Hi, 

Our Docker image is using the [CLI](https://mockoon.com/cli/) to run your mocks. The CLI does not expose a Swagger UI at the moment. This is rather the role of the desktop application which offers you a UI to design your mocks.
